src/screens/TopicMemberScreen/index.js

```diff
@@ -16,7 +9,6 @@ import dimen from '../../utils/dimen';
 import {getAllMemberTopic} from '../../service/topics';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
 import {colors} from '../../utils/colors';
-import Search from '../DiscoveryScreenV2/elements/Search';
 import StringConstant from '../../utils/string/StringConstant';
 import UsersFragment from '../DiscoveryScreenV2/fragment/UsersFragment';
 import {Context} from '../../context';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Search` from `../DiscoveryScreenV2/elements/Search` has been removed. If `Search` is not used elsewhere in the file, this change should be fine. However, if it's used without being imported, it will cause a runtime error.
```diff
- import Search from '../DiscoveryScreenV2/elements/Search';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->